electrolyzer/glue_code/run_lcoh.py

```diff
@@ -0,0 +1,240 @@
+"""
+This module is responsible for running electrolyzer models based on YAML configuration
+files.
+"""
+
+import numpy as np
+import pandas as pd
+
+import electrolyzer.inputs.validation as val
+from electrolyzer import LCOH  # ESG
+from electrolyzer import Supervisor
+
+from .optimization import calc_rated_system
+
+
+def _run_electrolyzer_full(modeling_options, power_signal):
```

Comment:
We might want to refactor this, as there's a lot of identical code to that in `run_electrolyzer`